### PR TITLE
check random provider in server

### PR DESCRIPTION
**Pull request by forgetso**

### CODE REVIEW

File: src/types/manager.ts [hunk 1]
```diff
@@ -43,15 +48,20 @@ export interface ProcaptchaState {
     account: Account | undefined // the account operating the challenge. undefined if not set
     submission: TCaptchaSubmitResult | undefined // the result of the captcha submission. undefined if not submitted
     timeout: NodeJS.Timeout | undefined // the timer for the captcha challenge. undefined if not set
+    blockNumber: u32 | undefined // the block number in which the random provider was chosen. undefined if not set
```
Comment (goastler) [inline src/types/manager.ts:51]: I thought block number was u64?

File: package.json [hunk 2]
```diff
@@ -14,15 +14,15 @@
   },
   "dependencies": {
     "@fingerprintjs/fingerprintjs": "^3.3.6",
-    "@polkadot/api": "^9.11.3",
-    "@polkadot/api-contract": "^9.11.3",
+    "@polkadot/api": "9.13.6",
```
Comment (goastler) [inline package.json:17]: Same again here, are we locking to polkadot versions?

**Review by goastler — CHANGES_REQUESTED** [review]

File: src/types/manager.ts (hunk 1 above)
Comment (forgetso) [reply src/types/manager.ts:51]: Not according to the ABI

File: package.json (hunk 2 above)
Comment (forgetso) [reply package.json:17]: Yes :)

**Review by goastler — APPROVED** [review]

**Pull request merged**